src/reconciler.tsx

```diff
@@ -196,7 +202,10 @@ export function applyProps(instance: any, newProps: any, oldProps: any = {}, acc
       if (container && instance.raycast) container.__interaction.push(instance)
       // Add handlers to the instances handler-map
       instance.__handlers = handlers.reduce(
-        (acc, key) => ({ ...acc, [key.charAt(2).toLowerCase() + key.substr(3)]: newProps[key] }),
```

Comment:
Inside of a reduce operation, it is safe/faster to mutate the accumulator rather than recreate + spread a new object each iteration.